tests/cms/views/link_replace/test_link_actions.py

```diff
@@ -0,0 +1,399 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from _pytest.logging import LogCaptureFixture
+    from pytest_django.fixtures import SettingsWrapper
+
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+from linkcheck.listeners import enable_listeners
+from linkcheck.models import Link, Url
+
+from integreat_cms.cms.models import Region
+from integreat_cms.cms.utils.linkcheck_utils import filter_urls
+from tests.conftest import ANONYMOUS, EDITOR, MANAGEMENT, STAFF_ROLES
+from tests.utils import assert_message_in_log
+
+# ----------- Test for url replace -----------#
+# Url to be replaced
+# This URL is used once in region Nürnberg and three times in Augsburg
+# If bulk action is conducted in the network management, all of them must be affected,
+# while only three of them if done in the broken link list of Augsburg
+OLD_URL = "https://integreat.app/augsburg/de/willkommen/"
+# Url to replace with
+NEW_URL = "https://integreat.app/augsburg/de/this-is-new-url/"
+# Parameters for test
+# (
+# <network management or region>,
+# <number of links with url=SEARCH_REPLACE_TARGET_URL before successful search&replace>,
+# <number of links with url=SEARCH_REPLACE_TARGET_URL which must stay unchanged after successful search&replace>
+# )
+url_replace_parameters = [("network_management", 4, 0), ("augsburg", 4, 1)]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("parameter", url_replace_parameters)
+def test_url_replace(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+    caplog: LogCaptureFixture,
+    parameter: tuple[str, int, int],
+) -> None:
+    """
+    Test whether link replacement in the network management per pencil icon in the column "options" work correctly.
+    """
+
+    settings.LANGUAGE_CODE = "en"
+    client, role = login_role_user
+    region, before, after = parameter
+
+    url_object = Url.objects.filter(url=OLD_URL).first()
+
+    assert Link.objects.filter(url=url_object).count() == before
+    assert not Link.objects.filter(url__url=NEW_URL).exists()
+
+    slugs = {
```

Comment:
```suggestion
    kwargs = {
```
Since those are not all slugs, maybe name it kwargs?